src/test/java/gov/usgs/wma/waterdata/BaseTestDao.java

```diff
@@ -55,6 +55,8 @@ public void setup() {
 		discreteGroundWater1.setFieldVisitIdentifier("0c8d1725-db51-47d9-aabb-a1dd9d1bdf08");
 		discreteGroundWater1.setLocationIdentifier(LOCATION_IDENTIFIER_1);
 		discreteGroundWater1.setAgencyCode("USGS");
+		discreteGroundWater1.setDateTimeAccuracyCode(DateTimeAccuracy.MINUTE.getCode());
+		discreteGroundWater1.setDateTimeAccuracyText(DateTimeAccuracy.MINUTE.getText());
```

Comment:
The sample dom object built up in this base test class are supposed to be after they have been loaded by the RowMapper, so they need to already be in a state where they follow the rules.